lib/components/viewers/stop-time-cell.tsx

```diff
@@ -89,42 +97,51 @@ const StopTimeCell = ({
   // Thursday 12:19am. We don't want the time to read: 'Thursday, 24 minutes'.
   const showDayOfWeek = !vehicleDepartsToday && !showCountdown
 
+  const realtime = stopTime.realtimeState === 'UPDATED'
   return (
-    <div>
-      <div className="pull-left">
-        <StyledIconWrapperTextAlign
-          style={{ color: '#888', fontSize: '0.8em', marginRight: 2 }}
-        >
-          {stopTime.realtimeState === 'UPDATED' ? <Rss /> : <Clock />}
-        </StyledIconWrapperTextAlign>
-      </div>
+    <>
+      <StyledIconWrapperTextAlign
+        style={{
+          fontSize: '0.6em',
+          margin: 0,
+          marginRight: 2,
+          transform: `scaleX(${realtime ? '-1' : '1'})`
```

Comment:
Move the `realtime` portion to the `PulsingRss` component and move the other styles to a styled wrapper?

Reply:
I think this would make it more difficult to a reader to understand what's going on

Reply:
See follow-up comment in `PulsingRss`, and move the non-transform style into a `styled(StyledIconWrapperTextAlign)`.